### PR TITLE
"Building ReactOS" belongs to "Building" part

### PR DESCRIPTION
**Pull request by jeditobe**
## Purpose

improved logic of the text

## Proposed changes

improved logic of the text
removed some line break

### CODE REVIEW

**Comment by SergeGautherie** [comment]
> removed some line break

Does that fix anything?

**Comment by jeditobe** [comment]
> > removed some line break
> 
> Does that fix anything?

it improves readability in browsers on some sizes of screens.

File: README.md [hunk 1]
```diff
@@ -74,9 +73,7 @@ By default, ReactOS currently can only be installed on a machine that has a FAT1
 The partition on which ReactOS is to be installed (which may or may not be the bootable partition) must also be formatted as FAT16 or FAT32.
 ReactOS Setup can format the partitions if needed.
 
-Starting 0.4.10, ReactOS can be installed using the BtrFS file system. But
-consider this as an experimental feature and thus regressions not triggered on
-FAT setup may be observed.
+Starting 0.4.10, ReactOS can be installed using the BtrFS file system. But consider this as an experimental feature and thus regressions not triggered on FAT setup may be observed.
```
Comment (Fraizeraust) [inline README.md:76]:
```suggestion
Starting with 0.4.10, ReactOS can be installed using the BtrFS file system. But consider this as an experimental feature and thus regressions not triggered on FAT setup may be observed.
```

**Comment by SergeGautherie** [comment]
> improves readability in browsers

Ah, it seemed to me EndOfLines were converted to Spaces, so made no difference...

**Pull request merged**